### PR TITLE
use directReplies

### PR DESCRIPTION
**Pull request by melisabok**
ECCV using this version in live site: https://openreview.net/group?id=thecvf.com/ECCV/2020/Conference/Program_Chairs

ICLR using this version in live site: https://openreview.net/group?id=ICLR.cc/2021/Conference/Program_Chairs

I think the improvement is significant. I would like to remove the call to get recommendation edges but it does too many calls or try to the get the edges grouped by signature.

### CODE REVIEW

**Review by zbialecki — COMMENTED** [review]
The code looks good to me, but while testing the ICLR 2021 homepage on live I got the follow JS error when I tried to do an export:

```
webfieldCode.js:2919 Uncaught TypeError: Cannot read property '1954' of undefined
```

In the `buildCSV` function it seems like the var `completedReviews` is undefined

**Comment by zbialecki** [comment]
Also, in the sorting seems to be broken on the Reviewer Status tab. 
```
webfieldCode.js:2001 Uncaught TypeError: Cannot read property 'numPapers' of undefined
    at Papers_with_Reviews_Missing (webfieldCode.js:2001)
```

File: openreview/conference/templates/programchairWebfield.js [hunk 1]
```diff
@@ -1703,20 +1806,40 @@ var displayAreaChairsStatusTable = function() {
...
-      return [number, summaryHtml, progressHtml, statusHtml];
+      if (SENIOR_AREA_CHAIRS_ID) {
+        return [number, summaryHtml, seniorSummaryHtml, progressHtml, statusHtml];;
+      } else {
+        return [number, summaryHtml, progressHtml, statusHtml];
+      }
     });
 
+    var headings;
+    if (SENIOR_AREA_CHAIRS_ID) {
```
Comment (melisabok) [inline openreview/conference/templates/programchairWebfield.js:1820]: here is the change @zbialecki

**Review by zbialecki — APPROVED** [review]
I think this can be merged too.

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**